helper-cli/src/main/kotlin/commands/curationshelper/CurationHelperCommand.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
```

Comment:
Please remove "helper" from command and package name and for "curations" please decide for plural or singular consistently. Using "PackageCuration" and "packagecuration" (package) would be my proposal.

Reply:
I renamed the the command and the package name.
The singular/plural choice for `create-package-curation` and `split-curations` was intentional as the first one creates a single file, the second one splits multiple curations into multiple files.

But I wouldn't care to adjust the command naming to singular if you think this is better for the usability.